src/core/dfn-abbr.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+// Module core/dfn
+// - Finds all <dfn> elements and populates conf.definitionMap to identify them.
+// - Also populates abbrMap from core/inlines for definitions that have an abbreviation
+
+import { registerDefinition } from "./dfn-map";
+
+export const name = "core/dfn-abbr";
+export const abbrMap = new Map();
+
+export function run() {
+  document.querySelectorAll("dfn[data-abbr]").forEach(dfn => {
+    if (dfn.dataset.abbr === "")
+      dfn.dataset.abbr = dfn.textContent
```

Comment:
This should be a function in itself, like `createAbbreviationFromText(textContent)`. Makes clearer what we're doing

Reply:
Done. Thanks!